woocommerce/woocommerce-ios
# Fix screenshot localization issues

This PR fixes an issue causing screenshots not to be localized. As a side-effect, it also dramatically speeds up screenshot generation by not recompiling the app for each locale/device pair.

It also changes the method we use to enter the password – when running in Russian, it would consistently fail if the simulator wasn't running in the foreground. 

**To Test**
1. Check out this PR
2. Run `bundle exec fastlane screenshots`
3. Ensure each generated screenshot set is in the expected language.

Update release notes:
